stories/layout/stack.js

```diff
@@ -44,7 +44,7 @@ export default function() {
   const sections = number('Sections', 2, { range: true, min: 2, max: 10 })
 
   return (
-    <Layout type="parent" className={style.page}>
+    <Layout type="parent" className={style.page} style={{ maxHeight: '100%' }}>
```

Comment:
after discussing with @arahansen, we'll put all css in a file and link it in the notes. That way it's obvious what's needed to generate the code for the page